backend/signing/configuration.go

```diff
@@ -89,6 +102,22 @@ func (configuration *Configuration) ExtendedPublicKeys() []*hdkeychain.ExtendedK
 	return configuration.extendedPublicKeys
 }
 
+// Address returns the configuration's address
+func (configuration *Configuration) Address() string {
+	return configuration.address
+}
+
+// IsAddressBased returns whether configuration is address based or not
+func (configuration *Configuration) IsAddressBased() bool {
+	if len(configuration.extendedPublicKeys) > 0 {
```

Comment:
Just `return configuration.address != ""` suffices, as the `New*` constructors don't produce the cases you test here.